### PR TITLE
I724 daylight saving

### PR DESCRIPTION
**Pull request by JIMENOFONSECA**
This PR cleans up the code so daylight saving hours are taking into account

### CODE REVIEW

File: cea/resources/radiation_arcgis/radiation.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 import datetime
 import os
+from datetime import timedelta
+from datetime import datetime as datetime2
```
Comment (daren-thomas) [inline cea/resources/radiation_arcgis/radiation.py]: this seems ugly... why not just use `datetime.timedelta` and `datetime.datetime`? a little more typing, but tacking on a `2` at the end of a variable feels weird to me...

File: cea/resources/radiation_arcgis/calculate_sunny_hours_of_year.py [hunk 2]
```diff
@@ -54,16 +54,11 @@ def calculate_sunny_hours_of_day(day, sunrise, temporary_folder):
...
-    if 90 <= day < 300:
-        D = 1
-    else:
-        D = 0
     # Calculation of Sunrise time
     Sunrise_time = sunrise[day - 1]
     # Calculation of table
     for x in range(values):
-        Hour = int(Sunrise_time) + int(D) + int(x)
+        Hour = int(Sunrise_time) + int(x)
```
Comment (daren-thomas) [inline cea/resources/radiation_arcgis/calculate_sunny_hours_of_year.py]: While not really a problem, it is slightly faster to not do the casting (`int(Sunrise_time)`):
```
In [4]: s = 3

In [5]: %%timeit
   ...: for x in range(1000):
   ...:    hour = s + x
   ...:
10000 loops, best of 3: 43.7 µs per loop

In [6]: %%timeit
   ...: for x in range(1000):
   ...:    hour = int(s) + int(x)
   ...:
1000 loops, best of 3: 246 µs per loop
```

File: cea/resources/radiation_arcgis/radiation.py [hunk 3]
```diff
@@ -366,6 +368,10 @@ def calculate_sunrise(year_to_simulate, longitude, latitude):
     print('complete calculating sunrise')
     return sunrise
 
+def is_dst(zonename):
```
Comment (daren-thomas) [inline cea/resources/radiation_arcgis/radiation.py]: this function is not used anywhere...

Comment (daren-thomas) [reply cea/resources/radiation_arcgis/radiation.py]: also, it doesn't have a comment? i'm just going to assume it returns a boolean, `True`, if `zonename` is wait... what?

File: cea/resources/radiation_arcgis/calculate_sunny_hours_of_year.py (hunk 2 above)
Comment (JIMENOFONSECA) [reply cea/resources/radiation_arcgis/calculate_sunny_hours_of_year.py]: it is needed, so better we keep it.

File: cea/resources/radiation_arcgis/radiation.py (hunk 1 above)
Comment (JIMENOFONSECA) [reply cea/resources/radiation_arcgis/radiation.py]: done

File: cea/resources/radiation_arcgis/radiation.py (hunk 3 above)
Comment (JIMENOFONSECA) [reply cea/resources/radiation_arcgis/radiation.py]: done

**Comment by JIMENOFONSECA** [comment]
@daren-thomas can we merge this?

**Pull request merged**